tests/test_graphical_units/test_geometry.py

```diff
@@ -168,6 +168,15 @@ def test_Angle(scene):
     scene.add(a)
 
 
+@frames_comparison
+def test_three_points_Angle(scene):
+    A = np.array([3, 5, 0])
+    B = np.array([0, 0, 0])
+    C = np.array([8, 0, 0])
+    a = Angle.from_three_points(A, B, C)
```

Comment:
It would be good to have a test where multiple cases for positions of the points A, B, C are constructed (what happens if A and C are close? Is the correct angle constructed when A and C swap?). It is not necessary to assign A B C to variables, the method can simply be called via `Angle.from_three_points([...], [...], [...])`.

Reply:
resolved